CriticalMassTests/ThemeControllerTests.swift

```diff
@@ -474,3 +474,15 @@ class ThemeControllerTests: XCTestCase {
         XCTAssertEqual(lightColor, Theme.light.style.titleTextColor)
     }
 }
+
+class MockThemeStore: ThemeStorable {
```

Comment:
This Store already exists in a separate file but it seems like the reference is lost

Reply:
true. had some issue during rebasing but forgot to resolve since it worked on my machine